src/components/posts/post-card.tsx

```diff
@@ -16,8 +16,8 @@ export const PostCard = (props: PostType) => {
     dispatch(updatePostLikes(id))
   }
 
-  let user = sampleUsers.find(user => user.id === props.user_id);
-  let recipe = recipesArray.find(recipe => recipe.id === props.recipe_id);
+  let user = sampleUsers.find(user => user._id === props.userId);
```

Comment:
If validation is completed we should revert to `recipe.id = props.recipeId` instead of hardcoding 8652

Reply:
this is gonna be replaced in the future w services, just for sake of debugging

Reply:
yeah this was a dirty fix to test rendering as well